CHANGELOG.rst

```diff
@@ -12,6 +12,15 @@ This project adheres to `Semantic Versioning`_ starting with version 0.11.0.
     https://github.com/RasaHQ/rasa-sdk/tree/master/changelog/ .
 .. towncrier release notes start
 
+[PR] - 2020-02-23
+^^^^^^^^^^^^^^^^^^^^
+
+Added
+-----
+- Copied over instance methods ``last_executed_action_has()``, ``get_last_event_for()`` and
+  ``applied_events`` from the Rasa ``DialogueStateTracker`` class to the SDK ``Tracker`` class.
+
+
```

Comment:
Read the note above, this file should not be edited directly. Please revert this change and add your entry based on instructions here: https://github.com/RasaHQ/rasa-sdk/tree/master/changelog